### DISCUSSION TITLE
Tile Upscaling => need older DML version to test

### DISCUSSION THREAD

**Discussion by ai-d-g** [lshqqytiger/stable-diffusion-webui-directml#323] [Q&A] [answered]
Apparently around June Automatic1111 had an update that broke the tile upscaling. I'm wondering if anyone knows which Directml would be from or before June that I can try and test if upscaling works. 

I'm not sure how to use  git to get to a previous version so a quick explanation would be awesome.

**Comment by ai-d-g** [comment, marked as answer]
I figured it out. This website is not the easiest to navigate ..

**Reply by Skyrion9** [reply]
It'd be helpful if you shared your findings.

**Reply by ai-d-g** [reply]
I tried a few different versions and I'm still getting seams in tiles even at 1024x1024. It takes a long time to get an image that looks decent enough to us Upscayl upscaler to get past 4096. It's most likely a combination of a AMD driver and the upscale models because the SD upscale doesn't have this issue but also don't do a great job improving image quality.
